src/unc_tools.cpp

```diff
@@ -160,7 +160,7 @@ void dump_out(unsigned int type)
          fprintf(D_file, "  orig_col %zu\n", pc->orig_col);
          fprintf(D_file, "  orig_col_end %zu\n", pc->orig_col_end);
          fprintf(D_file, (pc->orig_prev_sp != 0) ? "  orig_prev_sp %u\n" : "", pc->orig_prev_sp);
-         fprintf(D_file, (pc->flags != 0) ? "  flags %zu\n" : "", pc->flags);
+         fprintf(D_file, (pc->flags != 0) ? "  flags %llu\n" : "", pc->flags);
```

Comment:
Btw. we already handle `PRIx64` in [src/windows_compat.h](https://github.com/uncrustify/uncrustify/blob/5bf42800b0117a1f0451d56fffb2e5defb0a1d02/src/windows_compat.h#L29-L31) so I guess we could do the same for `PRIu64` / `llu`.
@codable could you update your PR?